app/decorators/case_contact_decorator.rb

```diff
@@ -25,7 +25,7 @@ def report_duration_minutes
   end
 
   def miles_traveled
-    object.miles_driven ? "#{object.miles_driven} 🚘" : "None"
+    (object.miles_driven.nil? || object.miles_driven == 0) ? "" : object.miles_driven 
```

Comment:
If you do what @littleforest suggests, this can become `object.miles_driven.zero?`